frontend/src/components/details/details.css

```diff
@@ -0,0 +1,105 @@
+.header{
+    font-size: 42px;
+    font-weight: 600;
+    margin-left: 0px;
+}
+
+.details_right{
+    margin-left: 600px;
+    margin-top: -400px;
+    width:50%;
+}
+
+.details_txt{
+    font-size: 20px;
+    margin: 0px;
+    margin-top: 15px;
+    width: 500px;
+    color: rgba(0, 0, 0, 0.6);
+}
+
+.list{
+    margin-top: 30px;
+    background-color: rgb(247, 163, 54, 0.7);
+    border-radius: 5px;
+    height: 160px;
+}
+
+ul{
+    margin-left: 40px;
+    font-size: 20px;
+    margin-top: 20px;
+    padding: 10px;
+}
+
+.book_a_slot{
+    width: 218px;
+    height: 62px;
+    background-color: rgb(255, 162, 57);
+    color: white;
+    font-size: 18px;
+    border-radius: 5px;
+    border-color: transparent;
+    margin-top: 40px;
+    font-weight: 900;
+    cursor: pointer;
+}
+
+.demo_video{
+    margin: 0;
+    padding: 0;
+    width: 35%;
+    height: 50%;
+    margin-top: -150px;
+    margin-left: 30px;
+    position: absolute;
+    top: 50%;
+}
+
+.details_left{
+    margin-top: 450px;
+    margin-left: 235px;
+}
+
+.bookaslot2{
+    background-color: rgb(255, 162, 57);
+    height: 85%;
+    width: 90%;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    display: flex;
+    top: 10%;
+    left: 5%;
+    color: white;
+    flex-direction: column;
+    font-size: 30px;
+    padding: 2px;
+    border-radius: 10px;
+}
+
+.calender{
+    width: 80%;
+    height: 50%;
+}
+
+.bookaslot3{
```

Comment:
Please convert the CSS selector to snake_case naming to match other selectors

Reply:
Done.